extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/AgroalRecorder.java

```diff
@@ -29,4 +35,24 @@ public AgroalDataSource get() {
         };
     }
 
+    public Supplier<DataSourcesExcludedFromHealthChecks> dataSourcesExcludedFromHealthChecks(
+            DataSourcesBuildTimeConfig dataSourcesBuildTimeConfig) {
+        return new Supplier<DataSourcesExcludedFromHealthChecks>() {
+            @Override
+            public DataSourcesExcludedFromHealthChecks get() {
+                List<String> excludedNames = new ArrayList<>();
+                if (dataSourcesBuildTimeConfig.defaultDataSource.healthCheckExclude) {
+                    excludedNames.add(null);
```

Comment:
Can we use a proper https://github.com/quarkusio/quarkus/blob/e963fe4f4887cdeaf0bdee3ae9f4040be0736a8b/extensions/datasource/common/src/main/java/io/quarkus/datasource/common/runtime/DataSourceUtil.java#L7 instead of `null`?

Reply:
done